### PR TITLE
Feature: KeyPair support

### PR DESCRIPTION
**Pull request by calvinlauco**
- Introduce `KeyPair` class for managing key pair
- `cro.KeyPair.makeRandom()`
  - Create a random Public Private key pair
- `cro.KeyPair.fromPublicKey(publicKey: Buffer)`
  - Create a KeyPair with the public key
- `cro.KeyPair.fromPrivateKey(privateKey: Buffer)`
  - Create a KeyPair with the private key and derived public key

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    "eslint-config-prettier": "^6.7.0",
+    "eslint-plugin-import": "^2.18.2",
+    "eslint-plugin-prettier": "^3.1.1",
+    "mocha": "^6.2.2",
+    "neon-cli": "^0.3.3",
+    "nodemon": "^1.19.4",
+    "prettier": "^1.19.1",
+    "sinon": "^7.5.0",
+    "ts-node": "^8.5.2",
+    "tslint": "^5.20.1",
```
Comment (CeruleanAtMonaco) [inline package.json]: tslint project is deprecating, I suggest to move to eslint fully

File: package.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+    "type": "git",
+    "url": "https://github.com/crypto-com/cro-nodelib"
+  },
+  "author": "Calvin Lau <calvin@crypto.com>",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "ow": "^0.15.0"
+  },
+  "devDependencies": {
+    "@types/chai": "^4.2.5",
```
Comment (CeruleanAtMonaco) [inline package.json]: no need for fixed version dependencies yet?

File: package.json (hunk 1 above)
Comment (calvinlauco) [reply package.json]: I think this tsline is an old dependency. I have switched to eslint already.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    "sinon": "^7.5.0",
+    "ts-node": "^8.5.2",
+    "tslint": "^5.20.1",
+    "typescript": "^3.7.2"
+  },
+  "scripts": {
+    "build": "neon build",
+    "nodemon": "nodemon",
+    "eslint": "eslint 'lib/**'",
+    "test": "mocha --require ./node_modules/ts-node/register --exit --color --recursive 'lib/src/**/*.spec.ts'"
```
Comment (CeruleanAtMonaco) [inline package.json]: I can't `yarn install` & `yarn test` successfully on node 13.2.0, yarn 1.19.1
what's the compatible node version? what did I miss?

**Comment by calvinlauco** [comment]
I just realize the project setup PR is force-closed. Please move discussion regarding project setup to https://github.com/crypto-com/cro-nodelib/pull/6

File: package.json (hunk 2 above)
Comment (calvinlauco) [reply package.json]: I will pin the dependencies in project setup.

**Pull request closed without merging**